preset/overrides.scss

```diff
@@ -1,5 +1,12 @@
 @import './variables.scss';
 
+@media (max-width: 588px) {
+  .v-expansion-panel-header__icon {
```

Comment:
Is this the type of change we're looking to add to all panel headers now and in the future?

Reply:
I've changed this now so there's a unique class added on the parent element, which get's the style we need when xsOnly here https://github.com/storypark/vue-cli-plugin-vuetify-preset-storypark/pull/10/commits/0934e1423a0ba0c87112bd261d7af94005ec33b2